code/SiteConfig.php

```diff
@@ -102,27 +102,45 @@ public function getCMSFields()
                 "Root",
                 $tabMain = new Tab(
                     'Main',
-                    $titleField = new TextField("Title", _t('SilverStripe\\SiteConfig\\SiteConfig.SITETITLE', "Site title")),
-                    $taglineField = new TextField("Tagline", _t('SilverStripe\\SiteConfig\\SiteConfig.SITETAGLINE', "Site Tagline/Slogan"))
+                    $titleField = new TextField("Title", _t(__CLASS__ . '.SITETITLE', "Site title")),
+                    $taglineField = new TextField("Tagline", _t(__CLASS__ . '.SITETAGLINE', "Site Tagline/Slogan"))
                 ),
                 $tabAccess = new Tab(
                     'Access',
-                    $viewersOptionsField = new OptionsetField("CanViewType", _t('SilverStripe\\SiteConfig\\SiteConfig.VIEWHEADER', "Who can view pages on this site?")),
-                    $viewerGroupsField = ListboxField::create("ViewerGroups", _t('SilverStripe\\CMS\\Model\\SiteTree.VIEWERGROUPS', "Viewer Groups"))
+                    $viewersOptionsField = new OptionsetField(
+                        "CanViewType",
+                        _t(__CLASS__ . '.VIEWHEADER', "Who can view pages on this site?")
+                    ),
+                    $viewerGroupsField = ListboxField::create(
+                        "ViewerGroups",
+                        _t('SilverStripe\\CMS\\Model\\SiteTree.VIEWERGROUPS', "Viewer Groups")
```

Comment:
You're using a mix of magic constants (`__CLASS__`) and strings `SilverStripe\\CMS\\Model\\SiteTree` to get FQCNs.

Please can you use a consistent style of `self::class` and `SiteTree::class`

Reply:
That was already the case - but I'll tidy up classes :-)